modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchIndexSearcher.java

```diff
@@ -169,6 +169,10 @@ else if (end < 0) {
 				}
 			}
```

Comment:
scroll up a few lines to 165... `_log.error(runtimeException,` runtimeException);`

during integration test execution this line is hit and another exception (java.lang.reflect.UndeclaredThrowableException) is thrown... not sure why